noir/noir-repo/acvm-repo/acir/src/circuit/opcodes.rs

```diff
@@ -29,6 +32,17 @@ pub enum Opcode {
         block_id: BlockId,
         init: Vec<Witness>,
     },
+    /// Calls to unconstrained functions
+    BrilligCall {
+        /// Inputs to the function call
+        inputs: Vec<BrilligInputs>,
+        /// Outputs to the function call
+        outputs: Vec<BrilligOutputs>,
+        /// A pointer to the index of the Brillig VM bytecode to be executed.
+        bytecode_index: u32,
+        /// Predicate of the Brillig execution - indicates if it should be skipped
+        predicate: Option<Expression>,
```

Comment:
```suggestion
        /// Id for the function being called. It is the responsibility of the executor
        /// to fetch the appropriate Brillig bytecode from this id.
        id: u32,
        /// Inputs to the function call
        inputs: Vec<BrilligInputs>,
        /// Outputs to the function call
        outputs: Vec<BrilligOutputs>,
        /// Predicate of the Brillig execution - indicates if it should be skipped
        predicate: Option<Expression>,
```

For consistency with `Opcode::Call`